scripts/backend_test_shards.json

```diff
@@ -361,6 +362,7 @@
         "core.domain.expression_parser_test",
         "core.domain.improvements_services_test",
         "core.domain.suggestion_registry_test",
+        "core.jobs.batch_jobs.suggestion_migration_jobs_test",
```

Comment:
I need to see passing backend tests so I can check runtime. Please re-assign me once the tests are passing

Reply:
DOne!

Reply:
The backend tests on the last commit [failed](https://github.com/oppia/oppia/actions/runs/3231091256/jobs/5290233145)